### DISCUSSION TITLE
Official VS Code plugin?

### DISCUSSION THREAD

**Discussion by mhkeller** [perspective-dev/perspective#2061] [Ideas]
Has the team considered creating an official integration to use a datagrid element in VS Code? It's a very nice experience to load a local csv and quickly analyze it with [this plugin](https://github.com/RandomFractals/vscode-data-preview/issues) but it hasn't been updated in a while and it would be great to use the new map feature. Would the team consider this integration?

**Comment by texodus** [comment]
I'm very interested in doing this - we had even offered in the past to maintain this project but it didn't work out.

We don't publish our roadmap (yet), but hope to post more details on our plans around this soon.
